support/Pipelines/Bbh/ControlId.py

```diff
@@ -15,7 +15,7 @@
 
 logger = logging.getLogger(__name__)
 
-DEFAULT_RESIDUAL_TOLERANCE = 1.0e-6
+DEFAULT_RESIDUAL_TOLERANCE = 1.0e-4
```

Comment:
Please add a comment here that this should be just below junk radiation effects on the controlled parameters

Reply:
Done!